README.md

```diff
@@ -202,6 +202,18 @@ by default) you can put the list of methods to add in a list that contains `all`
     - __call__
 ```
 
+You can create a code-block by referencing a file excerpt with `<literalinclude>` (sphinx-inspired) syntax. 
+```
+<literalinclude>
+{'path': './data/convert_literalinclude_dummy.txt', # relative path
+'language': 'python', # defaults to '' (empty str)
+'start-after': 'START python_import',
+'end-before': 'END python_import',
```

Comment:
Should those default to start and end of file, or always have to be provided?

Reply:
i'm happy to default them to `start and end of file`

Reply:
handled in https://github.com/huggingface/doc-builder/pull/179/commits/8bead14d4b1538ce68fd4a537dfa8ab240f99ede